recipes/pyjama/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pyjama" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "c1c9826fb2ad6b075858f97b2f82068085835bde28825d96028f753813fa2d74" %}

```

Found by using (on mac): `shasum -a 256 pyjama-1.0.0.tar.gz`
```suggestion
{% set sha256 = "32af0201765a28fdc96fec63dd7357156467c00c45dc8d52a529e3e31bc3febb" %}
```